submissions/Maksym-Perepelytsia/html-css-popup/index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Hooli-style Popup</title>
+    <link rel="icon" type="image/png" href="img/favicon.png">
+</head>
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <div class="header__logo">
+                <a href="/"><img src="img/logo.png" alt="Logo image"></a>
+            </div>
+            <nav>
+                <ul class="header__menu">
+                    <li class="header__menu__item">
+                        <a href="#" class="menu__link">Home</a>
+                    </li>
+                    <li class="header__menu__item">
+                        <a href="#" class="menu__link">News</a>
+                    </li>
+                    <li class="header__menu__item">
+                        <a href="#" class="menu__link">Contact</a>
+                    </li>
+                    <li class="header__menu__item popup-toggle">
+                        <input type="checkbox" id="popup-checkbox">
+                        <label for="popup-checkbox" class="popup-label">
+                            <img src="img/popup-button.png" alt="popup icon" width="24" height="24">
```

Comment:
For icons it's better set an empty alt attribute, read more [here](https://accessibility.psu.edu/images/imageshtml/#empty)